specs/derivation.md

```diff
@@ -987,11 +1060,19 @@ approximately 12 minutes, unless an attacker controls more than 1/3 of the total
 [merge]: https://ethereum.org/en/upgrades/merge/
 [l1-finality]: https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/#finality
 
-> **TODO** This was in the spec:
->
-> In practice, we'll pick an already-finalized L1 block as L2
-> inception point to preclude the possibility of a re-org past genesis, at the cost of a few empty blocks at the start
-> of the L2 chain.
->
-> This makes sense, but is in conflict with how the [L2 chain inception][g-l2-chain-inception] is currently determined,
-> which is via the L2 output oracle deployment & upgrades.
+# Constants
+
```

Comment:
I'm creating a linear issue to track that we need to settle on values for these and update the document.